### PR TITLE
added linkin project

### PR DESCRIPTION
**Pull request by RizkyRajitha**

### CODE REVIEW

File: _data/projects/linkin.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  - node.js
+  - platform
+  - postgres
+  - hacktoberfest
+  - linktree
+  - linktree-alternative
+  - vercel
+upforgrabs:
+  name: good first issue
+  link: https://github.com/RizkyRajitha/linkin/labels/good%20first%20issue
```
Comment (shiftkey) [inline _data/projects/linkin.yml:16]: :art: at the moment there are no issues found with this label, which means this won't be prioritized when this is merged. Were you planning to add any tasks to this label?

Comment (RizkyRajitha) [reply _data/projects/linkin.yml:16]: hi @shiftkey i added 2 issues for this repo . also i am planning to add more issues as well 

thank you

**Comment by RizkyRajitha** [comment]
@shiftkey kind reminder

**Comment by ritwik12** [comment]
@RizkyRajitha I'll merge this. looks like you've added the missing things

**Comment by ritwik12** [comment]
@RizkyRajitha Thanks!!

**Comment by RizkyRajitha** [comment]
Thank you @ritwik12

**Pull request merged**